roles/migrationcontroller/tasks/main.yml

```diff
@@ -433,9 +438,25 @@
         state: absent
         definition: "{{ lookup('template', 'controller.yml.j2') }}"
 
-    - name: "Set up mig controller"
+  - when: reconcile_migrationtenant
+    block:
+    #    set multi_tenant_controller_state present only when
+    #      it is reconcile_migration and migration_controller for the tenant is to be installed
```

Comment:
this got me to thinking, why is `migration_controller: true` field optional on MigrationTenant CR?

Reply:
No reason this needs to be here, let's remove it and revisit in another PR.